django/contrib/messages/constants.py

```diff
@@ -18,4 +20,5 @@
     "SUCCESS": SUCCESS,
     "WARNING": WARNING,
     "ERROR": ERROR,
+    "DANGER": danger
```

Comment:
Hello, How about "DANGER: DANGER" baesd on consistency.

Reply:
done in new commit